### PR TITLE
only show pytest panel if error

### PR DESCRIPTION
**Pull request by jgirardet**
status-bar blink green if success during 500ms

### CODE REVIEW

File: lib/atom-python-test.coffee [hunk 1]
```diff
@@ -58,6 +58,15 @@ module.exports = AtomPythonTest =
     exit = (code) ->
       atomPythonTestView = AtomPythonTest.atomPythonTestView
 
+      if atomPythonTestView.message.includes("success-line") #pytest retrun succes
+        statutbar = document.getElementsByClassName('status-bar')[0]
```
Comment (pghilardi) [inline lib/atom-python-test.coffee]: fix typo statubar to statusBar

**Comment by pghilardi** [comment]
@jgirardet ,

I am thinking and its better if this behavior may be configured in the plug-in settings too, so backward compatibility is not broke and maybe some users wants to see the panel on every run, even without failures.

So, could you add a new configuration named: "Only show test status panel on failures". The default value should be false the keep as it is today. If you need help on how to do this, I can help you.

Thanks again!

**Comment by jgirardet** [comment]
hello,
did you have time to check it ?

**Comment by pghilardi** [comment]
Sorry for the delay! I have fixed the conflicts and merged!

**Comment by jgirardet** [comment]
ok cool

**Pull request merged**